src/main/java/elements/TagImpl.java

```diff
@@ -0,0 +1,25 @@
+package elements;
+
+import org.awaitility.Awaitility;
+import org.openqa.selenium.StaleElementReferenceException;
+import org.openqa.selenium.WebElement;
+import util.ScrollUtils;
+import java.util.concurrent.TimeUnit;
+
+public class TagImpl extends ElementImpl implements Tag {
+
+    public TagImpl(WebElement element) {
+        super(element);
+    }
+
+    @Override
+    public void findTag() {
+        ScrollUtils.scrollUp(getWrappedElement());
+        try {
+            Awaitility.waitAtMost(5, TimeUnit.SECONDS)
```

Comment:
Why don't you use a Waiter here? why do you need awaitility?

Reply:
![image](https://user-images.githubusercontent.com/71931457/145395985-9c52d2c6-6922-407a-9455-bf87b8265e3e.png)

Reply:
fluent wait